content/posts/2018-01-10-kotlin-null.md

```diff
@@ -0,0 +1,249 @@
+---
+title: "Kotlin: The Problem with `null`"
+date: 2018-01-10
+slug: kotlin-the-problem-with-null
+---
+
+I think almost everybody in the CS world knows about the [Null References: The Billion Dollar Mistake](https://www.infoq.com/presentations/Null-References-The-Billion-Dollar-Mistake-Tony-Hoare) talk.
+However, it does not really matter if a concept of `null` references a good idea or a really bad one.
```

Comment:
"the concept" or just "concept"? "a" seems incorrect since you've defined it

Reply:
Yep, thanks.